src/js/utils/RouterUtil.js

```diff
@@ -186,6 +186,28 @@ const RouterUtil = {
       const blob = new Blob([data], { type: "application/json" });
       global.navigator.msSaveOrOpenBlob(blob, filename);
     }
+  },
+
+  /**
+   * Compares two urls if they have the same host name
+   *
+   * @param {String} urlA The first url
+   * @param {String} urlB The second url
+   * @returns {Boolean} - True if they have the same host, false if they don't
+   */
+  hasSameHost(urlA, urlB) {
```

Comment:
Shall we start thinking of a `UrlUtil`? I bet `RouterUtil` has been introduced to collect the `react-router` utils. Would be grate to extract all the Url related functions from other Utils and put them together. What do you think?